CHANGELOG.md

```diff
@@ -1,5 +1,181 @@
+
+#### iOS specific
+
+- Add asdf-vm support in find-node.sh ([3e7c310b1d](https://github.com/facebook/react-native/commit/3e7c310b1dcf5643920535eea70afa451888792a) by [@pastleo](https://github.com/pastleo))
+- - [iOS][fixed] - ScrollView: Respect `contentInset` when animating new items with `autoscrollToTopThreshold`, make `automaticallyAdjustKeyboardInsets` work with `autoscrollToTopThreshold` (includes vertical, vertical-inverted, horizontal and horizontal-inverted ScrollViews) ([6e903b07fa](https://github.com/facebook/react-native/commit/6e903b07fa8e8d9b78cae0e031bb8022f7a63195) by [@mrousavy](https://github.com/mrousavy))
+- Introduce MockSurfaceHandler ([ec92c85a15](https://github.com/facebook/react-native/commit/ec92c85a15468fd00dc0a23b8f69f5f1624f7b45) by [@philIip](https://github.com/philIip))
+
+### Changed
+
+- React Native sync for revisions e8feb11...afcb9cd ([c75ad2cdb1](https://github.com/facebook/react-native/commit/c75ad2cdb1b553af47dafb5840137d845b38be0b) by [@ShikaSD](https://github.com/ShikaSD))
+- Remove shared responsibility between LogBox and ExceptionsManager native module ([c901c43d11](https://github.com/facebook/react-native/commit/c901c43d11a243e7e4a9389f7e8f1440581ef7d9))
+- Update Modal's mock to not render its children when it is not visible ([ec614c16b3](https://github.com/facebook/react-native/commit/ec614c16b331bf3f793fda5780fa273d181a8492) by [@AntoineDoubovetzky](https://github.com/AntoineDoubovetzky))
+- Upgraded react-devtools-core dependency to 4.19.1 ([356236471a](https://github.com/facebook/react-native/commit/356236471abc6b5b8c139223e15388fd1eecd2d1) by [@jstejada](https://github.com/jstejada))
+- Changed the showcase URL ([1927ceb552](https://github.com/facebook/react-native/commit/1927ceb5520e36c5ca92e349ee1b860093290c15) by [@HeroBanana](https://github.com/HeroBanana))
+- ImageBackground now respects `imageStyle` width and height ([dbd5c3d8e5](https://github.com/facebook/react-native/commit/dbd5c3d8e5e35685be89156194a96cead553a330) by [@Naturalclar](https://github.com/Naturalclar))
+- React Native sync for revisions 95d762e...e8feb11 ([88021894f2](https://github.com/facebook/react-native/commit/88021894f2645266d7e1b0a2e61a737b1590d039) by [@kacieb](https://github.com/kacieb))
+- React Native sync for revisions bd5bf55...95d762e ([7f7f8188f2](https://github.com/facebook/react-native/commit/7f7f8188f2ed3cb40b85e7f1ad65fdf31b678ffc) by [@rickhanlonii](https://github.com/rickhanlonii))
+- Remove defaultProps from the DrawerLayoutAndroid Component.�lunaleaps this is the fix for issue https://github.com/facebook/react-native/issues/31606 ([b998e32501](https://github.com/facebook/react-native/commit/b998e32501ab13525a1a7c711d5ad090e3d23d9f) by [@mdaj06](https://github.com/mdaj06))
+- Bump Hermes to 0.9.0 ([8e66f0b35b](https://github.com/facebook/react-native/commit/8e66f0b35b5ec1054994f7048ef3e66002544197) by [@Huxpro](https://github.com/Huxpro))
```

Comment:
This was the highlight of 0.66.0 and is duplicated here, so should probably remove.